api/provider/aws/task_delete.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+// Delete stops a Task using the specified Task ID. The user's active cluster (Environment) is used as a filter
```

Comment:
Would we want to specify that a Task is an ECS Task?

Reply:
Yes, this should be explicitly specified.